File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/Lista de Exercícios 3/src/Ex17.java
//Importando Scanner
import java.util.Scanner;

public class Ex17 {

	public static void main(String[] args){
		
		//Instanciando Scanner
		Scanner ent = new Scanner(System.in);
		
		System.out.println("Soma dos Ímpares entre 1 e N\n\n");
		
		//vars
		int n, i, soma=0;
		
		//Entrada
		System.out.print("Digite um número N: ");
		n = ent.nextInt();
		
		//Processamento
		for( i=1; i<=n; i++ )
		{
			if( i%2 == 1 )
			{
				soma += i;
			}
		}
		
		//Saída
		System.out.println("Soma dos ímpares = " + soma);
		
	}
}

File: 3 semestre/Estrutura de Dados I/Listas/Lista de Matrizes/ex17.c
#include <stdio.h>

main()
{
    /* vars */
    int m[6][6], i, j,
        cont = 0;

    /* Entrada */
    for (i = 0; i < 6; i++) {
        for (j = 0; j < 6; j++) {
            printf("Digite m[%d][%d]: ", i, j);
            scanf("%d", &m[i][j]);
        }
    }

    /* Processamento */
    for (i = 0; i < 6; i++) {
        for (j = 0; j < 6; j++) {
            if (m[i][j] > 10) {
                cont++;
            }
        }
    }

    /* Saída */
    printf("\nVALORES MAIORES DO QUE 10: %d\n", cont);
}
